.github/workflows/drivers.yml

```diff
@@ -663,15 +663,13 @@ jobs:
       DRIVERS: vertica
     services:
       vertica:
-        image: sumitchawla/vertica
+        image: vertica/vertica-ce:12.0.2-0
         ports:
           - "5433:5433"
     steps:
     - uses: actions/checkout@v3
     - name: Make plugins directory
       run: mkdir plugins
-    - name: Fetch JDBC driver
-      run: wget --output-document=plugins/vertica-jdbc-7.1.2-0.jar ${{ secrets.VERTICA_JDBC_JAR }}
```

Comment:
I'm not quite sure why this was needed, the vertica plugin had a dependency on `com.vertica.jdbc/vertica-jdbc {:mvn/version "10.0.1-0"}`.

Reply:
At some point the Vertica driver was not available on Maven, I guess we forgot to remove this code once we were able to add the dependency